### PR TITLE
[MRG] FIX: plot_connectome displaying hemispheric cuts in single node color

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Fixes issue as reported in #1607 

This PR will allow to ```plot_connectome``` to display nodes when a unique color is passed in ```node_color``` parameter. It can be either be as a string node_color='red' or sequence ['red'].

Still-to-do

- Add into whats_new

Thank you for reviewing.

### CODE REVIEW

**Comment by banilo** [comment]
LGTM

File: doc/whats_new.rst [hunk 1]
```diff
@@ -21,6 +21,13 @@ Changes
       :class:`nilearn.decomposition.DictLearning` is deprecated and will
       be removed in next two releases. Use `components_img_` instead.
 
+Bug fixes
+---------
+
+    - Fix issues using :func:`nilearn.plotting.plot_connectome` when string is
+      passed in `node_color` display modes left and right hemispheric cuts
+      in the glass brain.
```
Comment (GaelVaroquaux) [inline doc/whats_new.rst]: It seems to me that this list is incomplete. We should add the other bug fixes before the release.

**Comment by GaelVaroquaux** [comment]
LGTM. I'll merge when CI passes. We'll fix the whats_new for the release.

**Comment by KamalakerDadi** [comment]
>We'll fix the whats_new for the release.

I can prepare that. Thanks

**Comment by bthirion** [comment]
LGTM.

**Pull request merged**